pkg/yttlibrary/assert.go

```diff
@@ -25,6 +27,30 @@ var (
 
 type assertModule struct{}
 
+func (b assertModule) Equals(thread *starlark.Thread, f *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if args.Len() != 2 {
+		return starlark.None, fmt.Errorf("expected two arguments")
+	}
+
+	expected := args.Index(0)
+	actual := args.Index(1)
+	if _, notOk := expected.(starlark.Callable); notOk {
+		return starlark.None, fmt.Errorf("lambdas are not allowed")
```

Comment:
Elsewhere in this file we use the word 'function' instead of 'lambda', so  similar wording like: "Expected argument not to be a function" would be more consistent

> 		return starlark.None, fmt.Errorf("expected argument to be a function, but was %T", lambda)